dashboard/app/controllers/foorm/forms_controller.rb

```diff
@@ -9,12 +9,12 @@ def update_questions
       questions_json = get_questions
       published_state = questions_json['published']
 
-      if published_state.nil_or_empty?
```

Comment:
Why did these things in this method change from `nil_or_empty`? and `to_bool` ?

Reply:
I had a bug here--I assumed the published state would come in as a string but it comes in as a boolean, so the extra checks weren't needed